azurerm/resource_arm_application_gateway.go

```diff
@@ -2657,6 +2698,29 @@ func flattenApplicationGatewayWafConfig(input *network.ApplicationGatewayWebAppl
 		output["max_request_body_size_kb"] = int(*input.MaxRequestBodySizeInKb)
 	}
 
+	exclusionLists := make([]interface{}, 0)
+	if exclusions := input.Exclusions; exclusions != nil {
+		for _, exclusionList := range *exclusions {
+			exclusionListOutput := map[string]interface{}{}
+
+			if exclusionList.MatchVariable != nil {
+				exclusionListOutput["match_variable"] = *exclusionList.MatchVariable
+			}
+
+			if exclusionList.SelectorMatchOperator != nil {
+				exclusionListOutput["selector_match_operator"] = *exclusionList.SelectorMatchOperator
+			}
+
+			if exclusionList.Selector != nil {
+				exclusionListOutput["selector"] = *exclusionList.Selector
+			}
+
+			exclusionLists = append(exclusionLists, exclusionListOutput)
+		}
+
+		output["exclusion"] = exclusionLists
```

Comment:
We want to have the above code in a flatten method, similar to what I suggested in the other review